apps/quickstart-integrations/README.md

```diff
@@ -56,15 +56,12 @@ In short, integrations run off of triggers that invoke certain actions. These tr
 1. Add your MONDAY_SIGNING_SECRET to .env file
    <br> \*\* To get your MONDAY_SIGNING_SECRET go to monday.com, open Developers section, open your app and find the Signing Secret in "Basic Information" section
    <br> ![Screenshot](https://dapulse-res.cloudinary.com/image/upload/f_auto,q_auto/remote_mondaycom_static/uploads/VladMystetskyi/4db4f03e-67a5-482d-893e-033db67ee09b_monday-Apps2020-05-1901-31-26.png)
-2. Run the server with ngrok tunnel with the command:
+2. Run the server using the monday tunnel:
 
 ```
 $ npm start
 ```
 
```

Comment:
Maybe we can write here where to find the public URL in the output of the console instead of the "open HTTP://localhost:4040/status to get your ngrok public URL"?